examples/reference/widgets/FileInput.ipynb

```diff
@@ -69,7 +80,32 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The widget also has a ``save`` method that allows saving the uploaded data to a file or [BytesIO](https://docs.python.org/3/library/io.html#binary-i-o) object."
+    "Similarly you can see the file name and mime type:"
```

Comment:
After uploading a file the `FileInput` disappears:

https://github.com/user-attachments/assets/98e185da-c3d0-4431-aa25-b3a2fd1f504d

Reply:
I believe this has now been solved. The problem is that I cannot test it because in Jupyter lab the cdn version 0.1.3 of panel-material-ui is loaded. Don't know how to avoid this.

![image](https://github.com/user-attachments/assets/771546fe-b1a4-4424-b11b-ac60576e4876)

How do I test in jupyterlab the current version @philippjfr ?